server/src/world/entity_scope_map.rs

```diff
@@ -20,13 +20,18 @@ impl<E: Copy + Eq + Hash> EntityScopeMap<E> {
         }
     }
 
-    pub fn get(&self, user_key: &UserKey, entity: &E) -> Option<&bool> {
+    pub fn get(&self, user_key: &UserKey, entity: &E) -> bool {
```

Comment:
I updated this function to return a bool instead of an option of a borrowed bool.
Let me know if there is a use-case for this function returning None instead of just false.